register.lic

```diff
@@ -0,0 +1,76 @@
+=begin
+  Search a deed register for the specified crafting material.
+  Output all lines that match.
+=end
+
+custom_require.call(%w[common common-items drinfomon])
+
+class Register
+  include DRC
+  include DRCI
+
+  def initialize
+    settings = get_settings
+	@search_results = []
+
+    arg_definitions =
+      [
+        [
+          { name: 'query', regex: /^[A-z\s\-\']+$/i, description: 'crafting material to search for, use quotes for multiple words' }
+        ]
+      ]
+
+    args = parse_args(arg_definitions)
+
+    if args.query
```

Comment:
Do you need to check `args.query` here? It is the only arg and not-optional.

Reply:
I guess not.